lib/data/data_source/remote/database_remote_source_impl.dart

```diff
@@ -13,11 +13,11 @@ import '../../dto/database/words_dto.dart';
 class DatabaseRemoteSourceImpl implements DatabaseRemoteSource {
   final database = FirebaseFirestore.instance;
   final userId = FirebaseAuth.instance.currentUser?.uid;
-
+//TODO: GETTING AND SETTING FOLDERS, HAVE TO BE CORRECTED
   @override
   Future<Success> newFolder(FolderDto dto) async {
     if (userId != null) {
-      await database.collection("$userId").doc(dto.folderName.toUpperCase()).set(dto.toJson());
+      await database.collection("users").doc("$userId").collection('userFolders').add(dto.toJson());
```

Comment:
```suggestion
      await database.collection("users").doc(userId).collection('userFolders').add(dto.toJson());
```
Move `"users"` to a private variable